core-bundle/src/Resources/contao/widgets/Picker.php

```diff
@@ -262,7 +262,13 @@ protected function renderLabel(array $arrRow, DataContainer $dc)
 		}
 
 		$labelConfig = &$GLOBALS['TL_DCA'][$dc->table]['list']['label'];
-		$label = vsprintf($labelConfig['format'], array_intersect_key($arrRow, array_flip($labelConfig['fields'])));
+		$labelValues = [];
+
+		foreach ($labelConfig['fields'] as $field) {
+			$labelValues[] = $arrRow[$field];
```

Comment:
Could the field be missing?

```suggestion
			$labelValues[] = $arrRow[$field] ?? '';
```

Reply:
Yes, it could. However, the previous implementation did not care for this either so I kinda went along with it 😂 
But I will improve this!